### PR TITLE
Week 06

### PR DESCRIPTION
**Pull request by manishtmtmt**

### CODE REVIEW

File: Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  displayWindowProperties();
+});
+
+function displayWindowProperties() {
+  // get the userAgent property from the window properties
+  const userAgent = window.navigator.userAgent;
+  console.log("displayWindowProperties ~ userAgent:", userAgent);
+
+  // get the screenWidth property from the window properties
+  const screenWidth = window.screen.width;
```
Comment (TanishShah37) [inline Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js]: an single const can be created for height and width both

File: Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  console.log("displayWindowProperties ~ screenWidth:", screenWidth);
+
+  // get the screenHeight property from the window properties
+  const screenHeight = window.screen.height;
+  console.log("displayWindowProperties ~ screenHeight:", screenHeight);
+
+  // get the currentUrl property from the window properties
+  const currentUrl = window.location.href;
```
Comment (TanishShah37) [inline Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js]: an single const can be created for currentURL and other location properties

**Review by TanishShah37 — CHANGES_REQUESTED** [review]

File: Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js (hunk 1 above)
Comment (manishtmtmt) [reply Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js]: const screenWidthAndHeight = `${window.screen.width} x ${window.screen.height}`;

File: Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js (hunk 2 above)
Comment (manishtmtmt) [reply Session-14_JS_DOM_Window_Object/Assignments/Mandatory/script.js]: const currentUrlAndPath = `${window.location.href} - ${window.location.pathname}`;

**Review by TanishShah37 — APPROVED** [review]

**Pull request merged**